misc/language-tour-guide/src/components/Loader.tsx

```diff
@@ -0,0 +1,13 @@
+interface LoaderProps {
+  message?: string;
+}
```

Comment:
We should use one solution, right now we have some props inline, this has interface and I think I saw `type` used as well.